setup.py

```diff
@@ -3,13 +3,17 @@
 with open('README.md') as readme_file:
     readme = readme_file.read()
 
+with open('requirements.txt') as requirements_file:
+    requirements = requirements_file.read().split('\n')
```

Comment:
i tried it and it works as if, but wanted to double check with you that we don't need to strip out the commented lines in `requirements.txt`
Currently, the list is:
```python
['black', 'datasets>=1.7.0', 'flake8', 'isort==5.8.0', 'pytest', 'pyyaml>=5', 'streamlit==0.82', 'jinja2', 'plotly', 'requests', 'pandas', '##############################################################', '# Dependencies in this section are added for specific datasets', '##############################################################', 'py7zr', '##############################################################', '# End of dataset-specific dependencies', '##############################################################', '']
```